spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateweekofyear">
+          <h1>CalendarDateWeekOfYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateWeekOfYear takes a date-like object _date_ and returns the week of the year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinweek">
+          <h1>CalendarDateDaysInWeek ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInWeek takes a date-like object _date_ and returns the number of days in the given week in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinmonth">
+          <h1>CalendarDateDaysInMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInMonth takes a date-like object _date_ and returns the number of days in the given month in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinyear">
+          <h1>CalendarDateDaysInYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateDaysInYear takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthsinyear">
+          <h1>CalendarDateMonthsInYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonthsInYear takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateinleapyear">
+          <h1>CalendarDateInLeapYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateInLeapYear takes a date-like object _date_ and returns *true* if it falls within a leap year for the calendar represented by _calendar_ and *false* otherwise.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefields">
+          <h1>CalendarDateFields ( _calendar_, _fields_ )</h1>
+          <p>The abstract operation CalendarDateFields takes a list of standard fields in _fields_ and returns a new list by adding relevant calendar-specific fields for the calendar represented by _calendar_.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemergefields">
+          <h1>CalendarDateMergeFields ( _calendar_, _fields_, _additionalFields_ )</h1>
+          <p>The abstract operation CalendarDateFields takes two lists of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new list that include both sets without any redudant fields.</p>
```

Comment:
```suggestion
          <p>The abstract operation CalendarDateFields takes two lists of calendar-specific fields for the calendar represented by _calendar_ in _fields_ and _additionalFields_ and returns a new list that include both sets without any redundant fields.</p>
```
(though I agree with Justin that we should probably add something about reconciling month and monthCode here, since that does apply to all calendars)